napari/layers/utils/color_transformations.py

```diff
@@ -3,18 +3,18 @@
 a numpy array with N rows, N being the number of data points, and a dtype of np.float32.
 
 """
-from itertools import cycle
-from typing import Union, List, Tuple, AnyStr
 import warnings
+from itertools import cycle
+from typing import AnyStr, List, Tuple, Union
 
-from vispy.color import Color, ColorArray
 import numpy as np
+from qtpy.QtGui import QColor
+from vispy.color import Color, ColorArray
 
 from ...utils.colormaps.standardize_color import transform_color
 
-
 # All parsable input datatypes that a user can provide
-ColorType = Union[List, Tuple, np.ndarray, AnyStr, Color, ColorArray]
+ColorType = Union[List, Tuple, np.ndarray, AnyStr, Color, ColorArray, QColor]
```

Comment:
Hmmm, I don't like this change... It sneaks Qt into our models code. It seems like the wrong solution. If you need a signature that takes in a QColor in addition to all of the above, I suggest defining a new type `QColorType = U[ColorType, QColor]` in our Qt files... If you want `standardize_color` to take in QColors, then I suggest modifying the dictionary from the view files.

Reply:
don't need it to.  I like the idea of pulling out all the qt stuff... just wasn't thinking.  Will update.